src/patches/managedPolicies.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { IAspect, aws_iam as iam } from 'aws-cdk-lib';
+import { IConstruct } from 'constructs';
+
+/**
+ * Patch for turning all Policies into ManagedPolicies
+ *
+ * Some customers have policies in place that make it impossible to create standard policies. Instead,
+ * they must use managed policies
+ *
+ * @example
```

Comment:
Can you include a note about how the ordering of Aspects added matters here?

i.e. If used in combination with _other_ Aspects in this project, this should most likely be added _before_ the others are added in.

Reply:
Added, let me know if that looks better